File: EmployeeBenefits.Data/Models/Dto/EmployeeDto.cs
namespace EmployeeBenefits.Data.Models.Dto
{
    public class EmployeeDto
    {
        public int Id { get; set; }

        public string? FirstName { get; set; }

        public string? LastName { get; set; }

        public decimal YearlySalary { get; set; }

        public decimal CheckGrossPay { get; set; }

        public decimal CostPerCheck { get; set; }

        public decimal YearlyCost { get; set; }

        public decimal Discounts { get; set; }

        public int Dependents { get; set; }
    }
}

File: EmployeeBenefits/ClientApp/src/app/components/employee-benefits/employeeDto.ts
export interface EmployeeDto {
    id:Number;
    firstName: string;
    lastName: string;
    dependents: DependentDto[];
    monthlyCost: Number;
    yearlyCost: Number;
}

export class Employee {
    id:Number = 0;
    firstName?: string;
    lastName?: string;
}

export interface DependentDto {
    id:Number;
    employeeId: Number;
    firstName: string;
    lastName: string;
}
